infra/build/functions/target_experiment.py

```diff
@@ -72,6 +72,7 @@ def run_experiment(project_name, target_name, args, output_path,
 
   build = build_project.Build('libfuzzer', 'address', 'x86_64')
   local_output_path = '/workspace/output.log'
+  local_err_path = '/workspace/err.log'
```

Comment:
nit: 
1. `local_jcc_err_path`
2. Add a comment to point this to `jcc.go`, e.g., `# From jcc.go:`